metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/entities/EntitiesController.java

```diff
@@ -103,15 +107,17 @@ public ResponseEntity<UrnResponseMap> getEntities(
 
   @PostMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<List<String>> postEntities(
-      @RequestBody @Nonnull List<UpsertAspectRequest> aspectRequests) {
+      @RequestBody @Nonnull List<UpsertAspectRequest> aspectRequests,
+      @RequestHeader(name = Constants.IN_UNMODIFIED_SINCE) String condUpdate) {
     log.info("INGEST PROPOSAL proposal: {}", aspectRequests);
 
     Authentication authentication = AuthenticationContext.getAuthentication();
     String actorUrnStr = authentication.getActor().toUrnStr();
 
+    Map<String, Long> createdOnMap = CondUpdateUtil.extractCondUpdate(condUpdate);
     List<Pair<String, Boolean>> responses = aspectRequests.stream()
         .map(MappingUtil::mapToProposal)
-        .map(proposal -> MappingUtil.ingestProposal(proposal, actorUrnStr,  _entityService, _objectMapper))
+        .map(proposal -> MappingUtil.ingestProposal(proposal, actorUrnStr,  _entityService, _objectMapper, createdOnMap.get(actorUrnStr)))
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/HTTP/Conditional_requests#avoiding_the_lost_update_problem_with_optimistic_locking, we will definitely need to return a [Precondition Failed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412) status so clients know to retry or adapt their request to the new state.

Are there any places we can do an integration test to show the behaviour from the client side?